### DISCUSSION TITLE
glob-parent vulnerability

### DISCUSSION THREAD

**Discussion by achilehero** [paulmillr/chokidar#1193] [Q&A] [answered]
Hello @paulmillr !

I noticed that there is this PR: https://github.com/paulmillr/chokidar/pull/1169 which was merged. Do you plan a release for that as well? I am having a vulnerability issue.

Thanks!

**Comment by paulmillr** [comment, marked as answer]
It was not merged. It was closed. There is no vulnerability. You must look into pinned issues before opening new ones.

**Reply by achilehero** [reply]
OK. Thanks for the clarification :)
